src/octopeer-github/content/EventTracker/MousePositionTracker.ts

```diff
@@ -44,8 +44,8 @@ class MousePositionTracker implements EventTracker {
             const windowObject = $(window);
             const viewportX = windowObject.scrollLeft();
             const viewportY = windowObject.scrollTop();
-            this.db.post(EventFactory.mousePosition(viewportX + this.x, viewportY + this.y,
-                viewportX, viewportY), EMPTY_CALLBACK, EMPTY_CALLBACK);
+            this.db.post(EventFactory.mousePosition(Math.round(viewportX + this.x), Math.round(viewportY + this.y),
+                Math.round(viewportX), Math.round(viewportY)), EMPTY_CALLBACK, EMPTY_CALLBACK);
```

Comment:
Why also round here when the coordinates are also rounded in the `RESTApiDatabaseAdapter`?